x-pack/plugins/enterprise_search/public/applications/enterprise_search/cypress.json

```diff
@@ -0,0 +1,21 @@
+{
+  "supportFile": false,
+  "pluginsFile": false,
+  "retries": {
+    "runMode": 2
+  },
+  "baseUrl": "http://localhost:5601",
+  "env": {
+    "username": "elastic",
+    "password": "changeme"
+  },
+  "fixturesFolder": false,
```

Comment:
The repeated `cypress.json` files may seem un-DRY, but IMO this separate per-product test suites will give individual teams more flexibility. For example, the Enterprise Search overview is much simpler and can set some folders/files to `false` unlike AS/WS.